docs/data/material/guides/next-js-app-router/next-js-app-router.md

```diff
@@ -0,0 +1,227 @@
+# Next.js App Router
+
+<p class="description">Learn how to use MUI libraries with the Next.js App Router</p>
+
+## Next.js and React Server Components
+
+Next.js's App Router implements React Server Components, which is a [new feature](https://github.com/reactjs/rfcs/blob/main/text/0227-server-module-conventions.md#changes-since-v1) in React.
+
+To support the App Router, currently all components and hooks from MUI libraries (Material UI, Joy UI, Base UI etc) are designated as client components and exported with the "use client" directive.
+
+:::info
+Don't need a guide? Use one of these example repos and get started:
```

Comment:
Rather than framing it in terms of need/skill, how about something like this?

```suggestion
Starting fresh on a new App Router-based project?
Jump right into the code with one of these example repos:
```